ghost/core/core/server/services/milestone-emails/index.js

```diff
@@ -0,0 +1,71 @@
+let hasScheduled = false;
+
+/**
+ * @param {'arr'|'members'} milestoneType
+ *
+ * @returns {Promise<any>}
+ */
+module.exports = async (milestoneType) => {
+    // const labs = require('../../../shared/labs');
+
+    // if (labs.isSet('milestoneEmails')) {
```

Comment:
@allouis I can't seem to access labs. It would always return `false` so not sure how to solve that.

Reply:
Note: I also tried requiring it in the job files with the same result

Reply:
Do you have `enableDeveloperExperiments` set? That's needed for alpha flags - also can you look at the `labs` setting and see what the value is

Reply:
Yup, all set. None of the flags are actually available at this stage it seems 😔